site/app/controllers/GlobalController.php

```diff
@@ -445,7 +447,7 @@ public function prep_user_sidebar(&$sidebar_buttons) {
 
         // --------------------------------------------------------------------------
         // INSTRUCTOR IN ANY COURSE
-        if (in_array($this->core->getUser()->getId(), $instructors)) {
+        if (!empty($this->core->getQueries()->getInstructorLevelAccessCourse($this->core->getUser()->getId()))) {
```

Comment:
No reason to run this query again if you can just store it in a variable from your query above on 394.